buildpack.md

```diff
@@ -117,58 +117,62 @@ The lifecycle MAY return an error to the platform if two or more buildpacks with
 
 ### Detection
 
-Executable: `/bin/detect <platform[AR]> <plan[E]>`, Working Dir: `<app[AR]>`
+Executable: `/bin/detect`, Working Dir: `<app[AR]>`
 
-| Input             | Description
-|-------------------|----------------------------------------------
-| `$0`              | Absolute path of `/bin/detect` executable
-| `<platform>/env/` | User-provided environment variables for build
-| `<platform>/#`    | Platform-specific extensions
+| Input                    | Attributes | Description                                   |
+|--------------------------|------------|-----------------------------------------------|
+| `$0`                     |            | Absolute path of `/bin/detect` executable     |
+| `$CNB_BUILD_PLAN_PATH`   | E          | Absolute path of the build plan               |
+| `$CNB_PLATFORM_DIR`      | AR         | Absolute path of the platform directory       |
```

Comment:
Maybe my browser is busted, but I'm not sure where that link points to, but I agree we should probably include buildpack dir here.

Reply:
Apologies, please see https://github.com/buildpacks/spec/pull/304 - if we merge that PR before we merge this one, it will have the effect intended by these suggestions.

Reply:
Is #304 intended to be merged as part of this?

Reply:
Yes, first 304 and then this one. I couldn't commit directly to the buildpack/0.8 branch.